lib/src/impl_js/impl_js.utils.dart

```diff
@@ -107,7 +107,7 @@ Future<T> _handleDomException<T>(
 }) async {
   try {
     return await fn();
-  } on subtle.DomException catch (e) {
+  } on subtle.JSDomException catch (e) {
```

Comment:
Will this work?

Reply:
Strange as it may seem, try-catch is possible..